src/ekka.app.src

```diff
@@ -1,6 +1,6 @@
 {application, ekka,
  [{description, "Autocluster for EMQ X Broker"},
-  {vsn, "0.13.1"},
+  {vsn, "0.13.5"},
```

Comment:
I note that `vsn` did not equal git tag for `0.13.4`, is this a mistake ? @zmstone